code/world.dm

```diff
@@ -17,4 +17,288 @@
 	view = "15x15"
 	hub = "Exadv1.spacestation13"
 	cache_lifespan = 0	//stops player uploaded stuff from being kept in the rsc past the current session
-	fps = 20
```

Comment:
note to self, check this against code/game/world.dm
changes here duped there, need to fix.